sw/device/silicon_creator/manuf/base/ft_personalize.c

```diff
@@ -446,7 +448,9 @@ static status_t personalize_gen_dice_certificates(ujson_t *uj) {
                             &cdi_0_key_ids, &curr_pubkey, all_certs,
                             &curr_cert_size));
   cdi_0_offset = perso_blob_to_host.next_free;
-  TRY(perso_tlv_prepare_cert_for_shipping("CDI_O", false, all_certs,
+  // DO NOT CHANGE THE "CDI_0" STRING BELOW with modifying the `dice_cert_names`
+  // collection in sw/host/provisioning/ft_lib/src/lib.rs.
+  TRY(perso_tlv_prepare_cert_for_shipping("CDI_0", false, all_certs,
```

Comment:
I meant to squash first and last patches of this stack  actually, but nevermind :)